tool/gravity/cli/app.go

```diff
@@ -181,11 +182,22 @@ func importApp(env *localenv.LocalEnvironment, registryURL, dockerURL, source st
 	}
 
 	if req.Vendor {
+		dockerClient, err := docker.NewDefaultClient()
```

Comment:
Should this consider `dockerURL` passed to this method? It's used above, when creating AppService.